frontend/src/views/Register.vue

```diff
@@ -57,17 +68,30 @@
   </v-container>
 </template>
 
+
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
 import config from '@/config'
 @Component
 export default class Register extends Vue {
   private passwordShow: boolean = false
+  private cPasswordShow: boolean = false
   private valid: boolean = false
   private username: string = ''
   private password: string = ''
+  private secretKey: string = ''
+  private accessKey: string = ''
   private register() {
-    const payload = { email: this.username, password: this.password }
+    const payload = { email: this.username, password: this.password,
+    secret_key: this.secretKey, access_key: this.accessKey}
+    fetch(config.apiUrl + '/register', {
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(payload),
+    }).then(() => this.$router.push('/'))
```

Comment:
You should maybe check that the result was successful.

Reply:
will do